src/main/java/org/broadinstitute/consent/http/db/DatasetDAO.java

```diff
@@ -108,102 +118,118 @@ Integer insertDataset(
+            + " LEFT OUTER JOIN dataset_property dp ON dp.dataset_id = d.dataset_id "
+            + " LEFT OUTER JOIN dictionary k ON k.key_id = dp.property_key "
+            + " LEFT OUTER JOIN consentassociations ca ON ca.datasetid = d.dataset_id "
             + " LEFT OUTER JOIN consents c ON c.consentid = ca.consentid "
-            + " ORDER BY d.datasetid ")
+            + " ORDER BY d.dataset_id ")
     Set<DatasetDTO> findAllDatasets();
 
     @Deprecated
     @UseRowMapper(DatasetDTOWithPropertiesMapper.class)
-    @SqlQuery("SELECT d.*, k.key, dp.propertyvalue, ca.consentid, c.dac_id, c.translateduserestriction " +
+    @SqlQuery(
+        "SELECT d.*, k.key, dp.property_value, ca.consentid, c.dac_id, c.translateduserestriction " +
           "FROM dataset d " +
-          "LEFT OUTER JOIN datasetproperty dp ON dp.datasetid = d.datasetid " +
-          "LEFT OUTER JOIN dictionary k ON k.keyid = dp.propertykey " +
-          "LEFT OUTER JOIN consentassociations ca ON ca.datasetid = d.datasetid " +
+          "LEFT OUTER JOIN dataset_property dp ON dp.dataset_id = d.dataset_id " +
+          "LEFT OUTER JOIN dictionary k ON k.key_id = dp.property_key " +
+          "LEFT OUTER JOIN consentassociations ca ON ca.datasetid = d.dataset_id " +
           "LEFT OUTER JOIN consents c ON c.consentid = ca.consentid " +
-          "WHERE d.datasetid = :datasetId ORDER BY d.datasetid, k.displayorder")
+          "WHERE d.dataset_id = :datasetId ORDER BY d.dataset_id, k.display_order")
     Set<DatasetDTO> findDatasetDTOWithPropertiesByDatasetId(@Bind("datasetId") Integer datasetId);
 
     @UseRowMapper(DatasetPropertyMapper.class)
-    @SqlQuery("SELECT * FROM datasetproperty WHERE datasetid = :datasetId")
+    @SqlQuery("SELECT * FROM dataset_property WHERE dataset_id = :datasetId")
     Set<DatasetProperty> findDatasetPropertiesByDatasetId(@Bind("datasetId") Integer datasetId);
 
     @Deprecated
     @UseRowMapper(DatasetDTOWithPropertiesMapper.class)
-    @SqlQuery(" SELECT d.*, k.key, dp.propertyvalue, ca.consentid, c.dac_id, c.translateduserestriction " +
-            " FROM dataset d INNER JOIN datasetproperty dp ON dp.datasetid = d.datasetid INNER JOIN dictionary k ON k.keyid = dp.propertykey " +
-            " INNER JOIN consentassociations ca ON ca.datasetid = d.datasetid INNER JOIN consents c ON c.consentid = ca.consentid " +
-            " WHERE d.datasetid IN (<dataSetIdList>) ORDER BY d.datasetid, k.receiveorder ")
+    @SqlQuery(
+        "SELECT d.*, k.key, dp.property_value, ca.consentid, c.dac_id, c.translateduserestriction " +
+            " FROM dataset d INNER JOIN dataset_property dp ON dp.dataset_id = d.dataset_id INNER JOIN dictionary k ON k.key_id = dp.property_key " +
+            " INNER JOIN consentassociations ca ON ca.datasetid = d.dataset_id INNER JOIN consents c ON c.consentid = ca.consentid " +
+            " WHERE d.dataset_id IN (<dataSetIdList>) ORDER BY d.dataset_id, k.receive_order ")
```

Comment:
Formatting could be improved here by adding newlines

Reply:
oops that happened with the merge, good catch!